packages/plugin-core/src/test/suite-integ/Extension.test.ts

```diff
@@ -169,77 +168,73 @@ suite("GIVEN SetupWorkspace Command", function () {
   let homeDirStub: SinonStub;
   let userConfigDirStub: SinonStub;
   let wsFoldersStub: SinonStub;
-
-  const ctx: ExtensionContext = setupBeforeAfter(this, {
-    beforeHook: async () => {
-      // Required for StateService Singleton Init at the moment.
-      // eslint-disable-next-line no-new
-      new StateService({
-        globalState: ctx.globalState,
-        workspaceState: ctx.workspaceState,
-      });
-      await resetCodeWorkspace();
-      await new ResetConfigCommand().execute({ scope: "all" });
-      homeDirStub = TestEngineUtils.mockHomeDir();
-      userConfigDirStub = mockUserConfigDir();
-      wsFoldersStub = stubWSFolders(undefined);
-    },
-    afterHook: async () => {
-      homeDirStub.restore();
-      userConfigDirStub.restore();
-      wsFoldersStub.restore();
-    },
-    noSetInstallStatus: true,
+  this.timeout(6 * 1000);
```

Comment:
can we pass this into `describreMultiWS` as a constant instead of setting it here?